### PR TITLE
feat(MeetingsAdapter): transform localAudio, localShare and localVideo into objects

### PR DESCRIPTION
**Pull request by alinasuciu**
Transform localVideo, localShare and localAudio into objects containing "stream" and "permission". Create a new object definition 

https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-249784

### CODE REVIEW

File: src/MeetingsAdapter.js [hunk 1]
```diff
@@ -16,14 +16,22 @@ import WebexAdapter from './WebexAdapter';
  * @see {@link https://developer.mozilla.org/en-US/docs/Web/API/MediaStreamTrack}
  */
 
+/**
+ * An object that represents a media stream and its permission.
+ *
+ * @typedef {object}  MeetingStream
+ * @property {MediaStream}       stream  The local media stream
+ * @property {MediaPermissions}  state   If meeting has permission to use media stream @see {@link MediaPermissions} enum
```
Comment (cipak) [inline src/MeetingsAdapter.js:24]: `state` -> `permission`

single space before between type and param name

File: src/MeetingsAdapter.js [hunk 2]
```diff
@@ -16,14 +16,22 @@ import WebexAdapter from './WebexAdapter';
...
+
 /**
  * A meeting object that allows users to have a WebRTC meeting.
  *
  * @typedef {object}  Meeting
  * @property {string}            ID           The meeting identifier
  * @property {string}            title        The title of the meeting
- * @property {MediaStreamTrack}  localVideo   The local video stream track
- * @property {MediaStreamTrack}  localAudio   The local audio stream track
+ * @property {MeetingStream}     localVideo   The local video stream
```
Comment (cipak) [inline src/MeetingsAdapter.js:33]: pls re-align the param names, one space (or more) between type and param name

File: src/MeetingsAdapter.js (hunk 1 above)
Comment (cipak) [reply src/MeetingsAdapter.js:24]: The state of the user permission on local media, @see ...

File: src/MeetingsAdapter.js [hunk 3]
```diff
@@ -16,19 +16,27 @@ import WebexAdapter from './WebexAdapter';
  * @see {@link https://developer.mozilla.org/en-US/docs/Web/API/MediaStreamTrack}
  */
 
+/**
+ * An object that represents a media stream and its permission.
```
Comment (lalli-flores) [inline src/MeetingsAdapter.js:20]:
```suggestion
 * An object that represents a media stream and its permission with respect to a meeting.
```

**Review by lalli-flores — COMMENTED** [review]
Can you remove the [`MediaStreamTrack`](https://github.com/webex/component-adapter-interfaces/pull/68/files#diff-e87864690545f91a837d21b65eb65c21d08b09ef73c338afdeac93d96f7eccd2L13) definition since it is no longer used?

**Comment by alinasuciu** [comment]
> Can you remove the [`MediaStreamTrack`](https://github.com/webex/component-adapter-interfaces/pull/68/files#diff-e87864690545f91a837d21b65eb65c21d08b09ef73c338afdeac93d96f7eccd2L13) definition since it is no longer used?

@lalli-flores, I made the requested changes and also added localShare on this pr

File: src/MeetingsAdapter.js [hunk 4]
```diff
@@ -10,25 +10,26 @@ import WebexAdapter from './WebexAdapter';
...
- * @property {MediaStream}       localShare   The local media share stream
- * @property {MediaStream}       remoteVideo  The remote video stream
- * @property {MediaStream}       remoteAudio  The remote audio stream
- * @property {MediaStream}       remoteShare  The remote media share stream
- * @property {MeetingState}      state        The state of the meeting
+ * @property {string}        ID           The meeting identifier
+ * @property {string}        title        The title of the meeting
+ * @property {MeetingStream} localVideo   The local video stream
+ * @property {MeetingStream} localAudio   The local audio stream
+ * @property {MeetingStream} localShare   The local share information
```
Comment (cipak) [inline src/MeetingsAdapter.js:28]: "stream" or "information" for all 3 please

**Review by lalli-flores — APPROVED** [review]

**Pull request merged**